_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post_longform
+title: Artsy's Technology Stack, 2017
+date: 2017-03-05
+categories: [Technology, eigen, force, gravity]
+author: [orta, artsy_engineering]
+series: Artsy Tech Stack
+---
+
+# History
+
+Artsy was [launched in 2012 as the "Art Genome Project"](http://www.nytimes.com/2012/10/09/arts/design/artsy-is-mapping-the-world-of-art-on-the-web.html) and grew exponentially ever since.
+
+By 2014 we had 230,000 works of art from 600 museums and institutions and launched our first business, a subscription service for commercial galleries, bringing over 80,000 works for sale and partnerships with 37 art fairs and a handful of benefit auctions. That year collectors from 82 countries inquired on over $5.5B of art.
+
+By 2015 we doubled our "for sale" inventory and aggregated 4,000 of the world's leading galleries and 60 art fairs. We also launched two new businesses: commercial auctions and online media.
+
+Finally, in 2016 we, again, doubled our paid gallery network size to become the largest gallery network in the world and grew to become the most-read online art publication as our highly engaging editorial traffic ballooned 320%. We also launched a platform to bid in live auctions and a consignments service with most major auction houses.
+
+
+# The Artsy Business in 2017
+
+In 2017 Artsy has three businesses at various stages of development and revenue: _Auctions_, _Content_ and _Listings_.
+
+* _Auctions_: Auction houses and charities use Artsy as a sales channel for a commission because collectors want to discover and buy art in a single, central platform that excels at surfacing the art they want from a global market.
+
+
+* _Content_: Brands pay Artsy to reach the first art audience at scale by enabling evergreen content online and for offline engagement during art world events.
+
+
+* _Listings_: Galleries, Fairs and Institutions subscribe to Artsy for a fee because we bring a very large audience of art collectors and enthusiasts to their virtual doors.
+
+The Artsy team is now 160 employees across three offices in New York, Berlin and London. The Engineering organization is now 28 engineers, including 4 leads, 3 directors and a CTO. In this post, we'd like to comprehensively cover what, and how we make the technical and human sides of Artsy businesses work.
```

Comment:
That number includes external contributors for Editorial/Genoming, but I bet this number has changed anyway - I'll ping Elisa and grab the latest, good thinkin